pkg/test/e2e/e2eutil/pravegacluster_util.go

```diff
@@ -890,21 +890,21 @@ func CheckStsExists(t *testing.T, f *framework.Framework, ctx *framework.TestCtx
 	return nil
 }
 
-func CheckConfigMapUpdated(t *testing.T, f *framework.Framework, ctx *framework.TestCtx, pravega *api.PravegaCluster, cmName string, key string, value string) error {
+func CheckConfigMapUpdated(t *testing.T, f *framework.Framework, ctx *framework.TestCtx, pravega *api.PravegaCluster, cmName string, key string, values []string) error {
 	cm := &corev1.ConfigMap{}
 	err := f.Client.Get(goctx.TODO(), types.NamespacedName{Namespace: pravega.Namespace, Name: cmName}, cm)
 	if err != nil {
 		return fmt.Errorf("failed to obtain configmap: %v", err)
 	}
 	if cm != nil {
 		optvalue := cm.Data[key]
-		if strings.Contains(optvalue, value) {
-			t.Logf("configmap is updated %s", cm.Name)
-			return nil
+		for _, value := range values {
+			if !strings.Contains(optvalue, value) {
+				return fmt.Errorf("config map is not updated")
+			}
 		}
 	}
```

Comment:
can we return error in case of config map is not found

Reply:
done